test/core/crawlers/basic_crawler.test.ts

```diff
@@ -1389,13 +1389,9 @@ describe('BasicCrawler', () => {
         const payload: Dictionary[] = [{ foo: 'bar', baz: 123 }];
         const getPayload: (id: string) => Dictionary[] = (id) => [{ foo: id }];
 
-        const tmpDir: string = `${__dirname}/tmp`;
+        const tmpDir: string = `${__dirname}/tmp/foo/bar`;
 
         beforeAll(async () => {
```

Comment:
This should be turned into afterAll

Reply:
i disagree, i want to have the folder for observation after the tests finish